webservice/src/main/java/com/hortonworks/streamline/webservice/StreamlineApplication.java

```diff
@@ -252,7 +253,7 @@ private void registerResources(StreamlineConfiguration configuration, Environmen
             authorizerConfig.put(DefaultStreamlineAuthorizer.CONF_CATALOG_SERVICE, securityCatalogService);
             authorizerConfig.put(DefaultStreamlineAuthorizer.CONF_ADMIN_PRINCIPALS, authorizerConf.getAdminPrincipals());
             authorizer.init(authorizerConfig);
-            environment.jersey().register(new StreamlineBasicAuthorizationRequestFilter());
+            environment.jersey().register(new StreamlineKerberosRequestFilter());
```

Comment:
This is at jax-rs level so will kick in only after all the servlet filters in the chain are executed.